tests/Makefile.am

```diff
@@ -271,40 +271,32 @@ XFAIL_TESTS +=					\
 	firstLineIndent.test			\
 	mixed_content.test			\
 	printPageNumberRange_03.test		\
-	mathml_nemeth/general_001.test		\
```

Comment:
Why did you remove all these tests?

Reply:
Because this tests already pass (@norbertmarkus doed required rule changes), and need remove this tests from xfail section in the tests/Makefile.am.
Now the xfail section removed already succesfull executed tests are ran right, 151 test passed from 219 tests.
Look following output part from test-suite.log when I ran make check command the tests directory:
```
================================================
   liblouisutdml 2.11.0: tests/test-suite.log
================================================

# TOTAL: 219
# PASS:  151
# SKIP:  0
# XFAIL: 68
# FAIL:  0
# XPASS: 0
# ERROR: 0
```
If we not remove the xpass tests (for example with you quoted your review), the tests are showed xpass status, and make stops with errors and build is stopping.

Attila

Reply:
Ah, OK, sorry. All fine then